readme.md

```diff
@@ -11,12 +11,28 @@
 ## Installing
 
 ```shell
-npm install derhuerst/asciicast-to-svg
+npm install -g derhuerst/asciicast-to-svg
```

Comment:
I'm constantly on the fence between: I'd like not to advocate for global installs, but using [npx](https://www.npmjs.com/package/npx) maybe be confusing for some people.

Reply:
Haha, yeah same for me. I didn't add it lightly ;-)